Eat_and_Greet/App.tsx

```diff
@@ -3,19 +3,35 @@ import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
 import HomeScreen from './src/screens/HomeScreen';
 import LoginScreen from './src/screens/LoginScreen';
-import OnboardingScreen from './src/screens/OnboardingScreen';
-
+import BasicInfo from './src/screens/OnboardingScreens/basicInfo';
+import Confirmation from './src/screens/OnboardingScreens/confirmation';
+import IdVerify from './src/screens/OnboardingScreens/idVerify';
+import SelectInterests from './src/screens/OnboardingScreens/selectInterests';
+import EmailVerify from './src/screens/OnboardingScreens/emailVerify';
 
 const Stack = createNativeStackNavigator();
+const OnboardingNavigation = createNativeStackNavigator();
 
 export default function App() {
   return (
     <NavigationContainer>
       <Stack.Navigator>
         <Stack.Screen name="Login" component={LoginScreen} />
         <Stack.Screen name="Home" component={HomeScreen} />
-        <Stack.Screen name="Onboarding" component={OnboardingScreen} />
+        <Stack.Screen name="Onboarding" component={OnboardingPagesNavigation} options={{ headerShown: true }}/>
       </Stack.Navigator>
     </NavigationContainer>
   );
 }
+
+function OnboardingPagesNavigation(){
+  return (
+    <OnboardingNavigation.Navigator>
+      <OnboardingNavigation.Screen name="Basic Info" component={BasicInfo} />
```

Comment:
Same note I gave rob, consider fixing to use a for loop iterating through an array of tuples (or similar idea) or create a ticket to remember to do this later.

Reply:
Done, made the ticket. 
https://github.com/apandey16/Meat_and_Feat/issues/36